libraries-2/src/main/java/com/baeldung/jcommander/usagebilling/service/DefaultFetchCurrentChargesService.java

```diff
@@ -0,0 +1,74 @@
+package com.baeldung.jcommander.usagebilling.service;
+
+import com.baeldung.jcommander.usagebilling.model.CurrentChargesRequest;
+import com.baeldung.jcommander.usagebilling.model.CurrentChargesResponse;
+import com.baeldung.jcommander.usagebilling.model.CurrentChargesResponse.LineItem;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static java.util.Arrays.asList;
+import static java.util.Arrays.fill;
+import static java.util.Collections.emptyList;
+import static java.util.concurrent.ThreadLocalRandom.current;
+import static java.util.stream.Collectors.toList;
+
+class DefaultFetchCurrentChargesService implements FetchCurrentChargesService {
+
+    DefaultFetchCurrentChargesService() {
+    }
```

Comment:
Why do we need a constructor? Won't we get one by default?

Reply:
Yes, missed this while cleaning up... was using a parameterized constructor in an earlier version of this class. Removed.